discord/commands/core.py

```diff
@@ -693,7 +693,7 @@ def _parse_options(self, params, *, check_params: bool = True) -> List[Option]:
             if option.default is None:
                 if p_obj.default == inspect.Parameter.empty:
                     option.default = None
-                elif issubclass(p_obj.default, (DiscordEnum, Enum)):
+                elif inspect.isclass(p_obj.default) and issubclass(p_obj.default, (DiscordEnum, Enum)):
```

Comment:
This should be faster as it removes a function call.
```suggestion
                elif isinstance(p_obj.default, type) and issubclass(p_obj.default, (DiscordEnum, Enum)):
```